src/main/java/com/iwaconsolti/school/miguel/controller/StudentController.java

```diff
@@ -0,0 +1,75 @@
+package com.iwaconsolti.school.miguel.controller;
+
+import com.iwaconsolti.school.miguel.model.Students;
+import com.iwaconsolti.school.miguel.model.dto.StudentsDTO;
+import com.iwaconsolti.school.miguel.service.StudentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/app/{nameSchool}")
+public class StudentController {
+
+    @Autowired
+    private StudentService studentService;
+
+    @PostMapping("/student")
+    public ResponseEntity<Object> registerStudent(@PathVariable String nameSchool, @RequestBody StudentsDTO studentDTO) {
+
+        if(studentDTO.getFirstName() == null || studentDTO.getLastName() == null || studentDTO.getAge() == 0){
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(
+                    Map.of("message","Required data is missing","status",HttpStatus.BAD_REQUEST.value())
+            );
+        } else if(studentService.getStudentById(nameSchool,studentDTO.getId()) != null){
```

Comment:
This can be a simple if, because if the previous if enter, the endpoint ends, because it has a return